libtatum/tatum/report/graphviz_dot_writer.cpp

```diff
@@ -89,8 +89,6 @@ void GraphvizDotWriter::write_dot_format(std::ostream& os,
                                          const std::map<NodeId,std::vector<TimingTag>>& node_tags,
                                          const std::map<NodeId,std::vector<TimingTag>>& node_slacks,
                                          TimingType timing_type) {
-    TATUM_ASSERT(timing_type == TimingType::SETUP || timing_type == TimingType::HOLD);
```

Comment:
@kmurray removed this because `GraphvizDotWriter::write_dot_file(std::string filename)` always fails.
Now that `TimingType::UNKOWN` is also handled, it can be allowed.